default/scripting/game_rules.focs.txt

```diff
@@ -133,3 +133,12 @@ GameRule
     category = "MULTIPLAYER"
     type = Toggle
     default = Off
+
+GameRule
```

Comment:
I see one game rule, should we have 2 of them?

Reply:
Actually i am not using short range targeting yet, (the other targeting of tentacle depends on when fighters launch, which is not scriptable in focs). If there is an instance of shortrange weapon i will add it.

Reply:
Please remove string tables entries: RULE_FIRST_COMBAT_ROUND_IN_SHORT_TARGETING_RANGE*

Reply:
done